README.md

```diff
@@ -1,2 +1,74 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# OAM Kubernetes Runtime
```

Comment:
A Github badge just a link to https://pkg.go.dev/mod/github.com/crossplane/oam-kubernetes-runtime may help folks discover the API docs.

Reply:
what is a Github badge?